plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduClientSession.java

```diff
@@ -436,6 +430,18 @@ private void setTypeAttributes(ColumnMetadata columnMetadata, ColumnSchemaBuilde
         }
     }
 
+    private ColumnSchema addNullableColumnSchema(ColumnMetadata columnMetadata)
```

Comment:
I would inline this method as it's simple and not reused.

Reply:
done~